### PR TITLE
Ember 3.20

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,23 +31,23 @@
...
   "dependencies": {
-    "ember-cli-babel": "^7.19.0",
-    "ember-cli-htmlbars": "^5.0.0",
+    "ember-cli-babel": "^7.21.0",
+    "ember-cli-htmlbars": "^5.2.0",
     "ember-cli-node-assets": "^0.2.2",
     "ember-diff-attrs": "^0.2.2",
     "fastboot-transform": "^0.1.3",
-    "flatpickr": "^4.6.3"
+    "flatpickr": "4.6.3"
```
Comment (Turbo87) [inline package.json:39]: why did you pin the version? seems unrelated to the Ember update, no?

Comment (rwwagner90) [reply package.json:39]: Pinned until I have time to debug why latest makes things fail

Comment (Turbo87) [reply package.json:39]: I've been using 4.6.5 and it seems to work fine for me 🤔

Comment (rwwagner90) [reply package.json:39]: 4.6.6 was causing failures both here and in Ember Inspector. Flatpickr is notorious for releasing a patch version and changing the entire internal structure of the classes, mousedown vs click etc handlers, but only releasing a patch because the external API did not change. When we write tests against it though, they all start failing and we have to figure out what the new thing is.

Comment (Turbo87) [reply package.json:39]: I see, thanks. might be worth mentioning in the changelog though 😉

Comment (rwwagner90) [reply package.json:39]: Yes, for sure. The hope is to get a full refactor away from mixins in shortly, update flatpickr to latest, and get a new major release out. Apologies for the temporary lack of detail on the change, but stay tuned 😃
